file.go

```diff
@@ -38,7 +36,7 @@ func MustCreate(fileanme string) io.WriteCloser {
 //            return s
 //    }).(*AStruct)
 func OpenAnd(file string, fn func(r io.Reader) interface{}) interface{} {
```

Comment:
It is a Go convention since Go standard package `os` provides `os.Open` to open a file for reading, and `os.Create` to open (create) a file only for writing. How about rename `OpenAnd` to be `WithOpened` and rename `CreateAnd` to be `WithCreated`?